docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -25,6 +35,16 @@ appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+The ``connection-url`` defines the connection URL string and parameters to pass
```

Comment:
The parameters are part of the URL string. You could say "connection information and parameters"

Reply:
updated